### PR TITLE
LRQA-72987 - Workflow Automated Test CanNotSaveProcessWithEmptyNodeID

### PR DESCRIPTION
**Pull request by alanafergs**
https://issues.liferay.com/browse/LRQA-72987

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase [hunk 1]
```diff
@@ -392,11 +392,26 @@ definition {
...
+			nodeType = "task");
+
+		ProcessBuilderKaleoDesignerReact.editNodeInformationField(
+			infoOption = "Id",
+			infoValue = "");
 
-		// TODO LRQA-72987 CanNotSaveProcessWithEmptyNodeID pending implementation
+		ProcessBuilderKaleoDesignerReact.viewRequiredField(
+			requiredField = "nodeId",
+			save = "true");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase]: Step 5 is missing. Assert the message "Some fields need to be filled.Please fill out the fields before saving or publishing."

**Review by lucasperj — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase (hunk 1 above)
Comment (alanafergs) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase]: The Assert this included in the macro `viewRequiredField`

**Review by lucasperj — APPROVED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase (hunk 1 above)
Comment (lucasperj) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase]: :+1:

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:forward

**Pull request closed without merging**